config/imsim-config.yaml

```diff
@@ -39,6 +39,9 @@ eval_variables:
         type: Degrees
         theta: { type: OpsimData, field: rotTelPos }
 
+    fpupil_R_outer: 4.18
+    fpupil_R_inner: 2.55  # M1 inner diameter is 2.558, but we need a bit of slack for off-axis rays
+
```

Comment:
Maybe just put `fpupil_area` here once rather than duplicating the code?

Reply:
Sure, I had thought that we might want to access those values individually, at some point, but the skycat and instcat implementations indeed probably don't need that.